invenio_app_ils/circulation/search.py

```diff
@@ -138,3 +138,10 @@ def get_loans_by_patron_pid(patron_pid):
         "term", patron_pid=patron_pid
     )
     return search
+
+
+def get_active_loans_by_patron_pid(patron_pid):
+    """Returns all the ongoing loans for a given patron."""
```

Comment:
minor: Maybe a small refactoring in the docstring as it doesn't return the results i.e the ongoing loans: 
```suggestion
    """Constructs the search object for all the ongoing loans of a given patron."""
```
@kprzerwa WDYT?